sdk/storage/azure-storage-blob/azure/storage/blob/_serialize.py

```diff
@@ -85,17 +85,15 @@ def _get_match_headers(kwargs, match_param, etag_param):
     return if_match, if_none_match
 
 
-def get_access_conditions(lease):
-    # type: (Optional[Union[BlobLeaseClient, str]]) -> Union[LeaseAccessConditions, None]
+def get_access_conditions(lease: Optional[Union["BlobLeaseClient", str]]) -> Union[LeaseAccessConditions, None]:
```

Comment:
Union[LeaseAccessConditions, None] -> Optional[LeaseAccessConditions]?

Reply:
I was so confused what I was looking at until I realized the arrow `->` just meant you want me to change it 😄